src/systems/si/include/units/si/constants.h

```diff
@@ -0,0 +1,71 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2018 Mateusz Pusz
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#pragma once
+
+#include <units/isq/mechanics.h>
+#include <units/isq/space_and_time.h>
+#include <units/isq/thermodynamics.h>
+#include <units/quantity.h>
+#include <units/si/units.h>
+
+namespace units::si {
+
+namespace si2019 {
+
+// clang-format off
+inline constexpr struct hyperfine_structure_transition_frequency_of_cs_unit :
+  constant_unit<basic_symbol_text{"ΔνCs" "dvCs"}, mag<9'192'631'770> * hertz> {} hyperfine_structure_transition_frequency_of_cs_unit;
+inline constexpr struct speed_of_light_in_vacuum_unit :
+  constant_unit<"c", mag<299'792'458> * metre / second> {} speed_of_light_in_vacuum_unit;
+inline constexpr struct planck_constant_unit :
+  constant_unit<"h", mag<ratio{662'607'015, 100'000'000}> * mag_power<10, -34> * joule * second> {} planck_constant_unit;
+inline constexpr struct elementary_charge_unit :
+  constant_unit<"e", mag<ratio{1'602'176'634, 1'000'000'000}> * mag_power<10, -19> * coulomb> {} elementary_charge_unit;
```

Comment:
Does `quantity_cast<electric_charge<coulomb>>(electric_charge<elementary_charge_unit>(5))` do `5 * 1.602'176'634e-19`, as before, or something more like `5.0 * 1'602'176'634 / 1'000'000'000 * pow(10, -19)`?

Reply:
I am on vacation now and can't check, but I think a bit of both. See here:

https://github.com/mpusz/units/blob/b4828d243d9332a17e406e4c0c9767811b5870b6/src/core/include/units/quantity_cast.h#L94-L104

Reply:
I am back from vacation. Do you see a problem here?

Reply:
Not for this change, which uses `constant_unit`, new in V2, where before it was a constant of value `1.602'176'634e-19`.

Reply:
This feature tries to address #169, and I generally think it is a good idea as in many cases physical constants simplify themselves (multiplication and division by the same constant value). Also, it allows us to specify quantities as being ratios of constants (i.e. `0.5 [c]` for speed).